browser/ui/views/brave_tooltips/BUILD.gn

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//build/config/features.gni")
+import("//build/config/ui.gni")
+import("//testing/test.gni")
+import("//ui/base/ui_features.gni")
+
+assert(is_win || is_linux || is_mac)
```

Comment:
assert(brave_adaptive_captcha_enabled) ?

Reply:
My plan is that `brave_tooltips` will be more of a generic component. So while `brave_adaptive_captcha` does use tooltips, I don't necessarily want the tooltips component to know anything about `brave_adaptive_captcha` (or other components that use it). Does that make sense?

Reply:
Got it. I assumed adaptive_captcha is the only user of this.